marrow/mailer/message.py

```diff
@@ -258,7 +258,8 @@ def mime(self):
 		return message
 
 	def attach(self, name, data=None, maintype=None, subtype=None,
-		inline=False, filename=None, encoding=None):
+		inline=False, filename=None, filename_charset='', filename_language='',
+		content_encoding=None):
```

Comment:
Oops! You're totally right. My goal was to clarify the difference between the encoding of the attachment and the encoding of the attachment name. However, RFC uses "charset" rather than "encoding".

If that is your only concern, I'd be happy to push that change up right now. Otherwise, I'll bundle all of my changes into another commit for easier reviewing.